plugins/securityPolicies/ua_securitypolicies.h

```diff
@@ -34,6 +35,12 @@ UA_SecurityPolicy_Basic256Sha256(UA_SecurityPolicy *policy,
                                  const UA_ByteString localPrivateKey,
                                  const UA_Logger *logger);
 
+/* Internal definitions for reuse between policies */
+UA_StatusCode
+generateKey_sha1p(mbedtls_md_context_t *sha1MdContext,
```

Comment:
We don't need to expose the mbedtls specific context here.
It suffices to allocate an md_context inside the generate key function and then deallocate it again.
That way the generateKey function can be generic and agnostic from the library that is used to implement it.

Reply:
Moved to an internal "common" header.